internal/cephfs/controllerserver.go

```diff
@@ -185,7 +185,12 @@ func (cs *ControllerServer) CreateVolume(
 	defer volOptions.Destroy()
 
 	if req.GetCapacityRange() != nil {
-		volOptions.Size = util.RoundOffBytes(req.GetCapacityRange().GetRequiredBytes())
+		// we were doing round off ( by util.RoundOffBytes) to the size passed
+		// in from the provisioner before ceph csi v3.4.0, however this can
+		// cause wastage of space in the cluster. The CephFS filesystem is capable
+		// of creating volume with the exact size passed in. so no need of Roundoff
+		// here before we reach out to the cluster
+		volOptions.Size = req.GetCapacityRange().GetRequiredBytes()
```

Comment:
There might be an issue where a volume was created with a previous version, and a new volume cloned from the original volume with the same size given? If  the original volume's size was rounded up, and the clone request contains a smaller size (not round-up), will that work?

Reply:
Shouldnt be an issue as the clone keep the parent size.  I have also explained this in the commit message.